project/build.scala

```diff
@@ -315,17 +320,22 @@ object Dependency {
   val jerseyMultiPart =  "com.sun.jersey.contribs" % "jersey-multipart" % V.Jersey
   val jodaTime = "joda-time" % "joda-time" % V.JodaTime
   val jodaConvert = "org.joda" % "joda-convert" % V.JodaConvert
-  val twitterCommons = "com.twitter.common.zookeeper" % "candidate" % V.TwitterCommons
-  val uuidGenerator = "com.fasterxml.uuid" % "java-uuid-generator" % V.UUIDGenerator
+  val twitterCommons = "com.twitter.common.zookeeper" % "candidate" % V.TwitterCommons excludeAll (excludeSlf4jLog4j12,
+    excludeLog4j)
+  val uuidGenerator = "com.fasterxml.uuid" % "java-uuid-generator" % V.UUIDGenerator excludeAll excludeLog4j
   val jGraphT = "org.javabits.jgrapht" % "jgrapht-core" % V.JGraphT
-  val hadoopHdfs = "org.apache.hadoop" % "hadoop-hdfs" % V.Hadoop excludeAll(excludeMortbayJetty, excludeJavaxServlet)
-  val hadoopCommon = "org.apache.hadoop" % "hadoop-common" % V.Hadoop excludeAll(excludeMortbayJetty, excludeJavaxServlet)
+  val hadoopHdfs = "org.apache.hadoop" % "hadoop-hdfs" % V.Hadoop excludeAll(excludeMortbayJetty, excludeJavaxServlet,
+    excludeLog4j)
+  val hadoopCommon = "org.apache.hadoop" % "hadoop-common" % V.Hadoop excludeAll(excludeMortbayJetty,
+    excludeJavaxServlet, excludeSlf4jLog4j12, excludeLog4j)
   val beanUtils = "commons-beanutils" % "commons-beanutils" % "1.9.2"
   val scallop = "org.rogach" %% "scallop" % V.Scallop
   val jsonSchemaValidator = "com.github.fge" % "json-schema-validator" % V.JsonSchemaValidator
-  val twitterZk = "com.twitter" %% "util-zk" % V.TwitterZk
+  val twitterZk = "com.twitter" %% "util-zk" % V.TwitterZk excludeAll(excludeSlf4jLog4j12, excludeLog4j)
   val rxScala = "io.reactivex" %% "rxscala" % V.RxScala
   val marathonUI = "mesosphere.marathon" % "ui" % V.MarathonUI
+  val logback = "ch.qos.logback" % "logback-classic" % V.Logback
+  val log4jOverSlf4j = "org.slf4j" % "log4j-over-slf4j" % V.Log4jOverSlf4j
```

Comment:
Add bridges for all default logger frameworks:
      val Slf4jJul = "org.slf4j" % "jul-to-slf4j" % Version.Slf4j % "compile"
      val Slf4jJcl = "org.slf4j" % "jcl-over-slf4j" % Version.Slf4j % "compile"
      val Slf4jLog4j = "org.slf4j" % "log4j-over-slf4j" % Version.Slf4j % "compile"